Src/BootCamp.Chapter/LINQExtension.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System.Collections.Generic;
+
+namespace BootCamp.Chapter
+{
+    public static class LINQExtension
+    {
+        public static IEnumerable<T> SnapFingers<T>(this IEnumerable<T> source)
+        {
+            List<T> list = new List<T>();
+
+            int i = 0;
+
+            foreach (var element in source)
```

Comment:
This is ``source.Count()``

Reply:
You should have used LINQ. Also there is no random factor what gets selected- always the first half.

Reply:
Can you implement a select using LINQ :) ?

Reply:
woeps forgot about this.  missed the using LINQ thought it was weird i could not use Count().
can you please re review it for me thanks kai!